example/test_problem/Hydro/CCSN/Input__Flag_User.CoreCollapse

```diff
@@ -1,10 +1,10 @@
 # Level                Density
-      0                3.0e-9
-      1                1.0e-8
-      2                3.0e-8
-      3                1.0e-7
-      4                3.0e-7
-      5                3.0e-7
-      6                3.0e-7
-      7                3.0e-7
-      8                3.0e-7
+      0                    0.0 
+      1                    0.0 
+      2                    0.0 
+      3                    0.0 
+      4                    0.0 
+      5                    0.0 
+      6                    0.0 
+      7                    0.0 
+      8                    0.0 
```

Comment:
Remove the trailing spaces in `Input__Flag_User.*`, but keep the empty line at the end of files.

Reply:
I have removed the spaces.